### PR TITLE
Update impress.js

### PR DESCRIPTION
**Pull request by EasonWang01**
Enable mouse wheel to change current view.

### CODE REVIEW

**Comment by EasonWang01** [comment]
1.Add mouse wheel event to change page.
2.Add H button to able redirect to overview page.

File: js/impress.js [hunk 1]
```diff
@@ -703,6 +703,12 @@
         //   consistency I should add [shift+tab] as opposite action...
         document.addEventListener("keyup", function ( event ) {
 
+            ///press H button to redirect to the overview page,if your overview changes,
+            //just change the words after #/ for what your overview page's id.
+            if(event.keyCode==72){
+                window.location.replace(window.location.pathname+"#/overview");
```
Comment (FagnerMartinsBrack) [inline js/impress.js]: We can't couple `impress.js` with the reference of an external URL. The "overview" from the impress.js example is application specific and not mandatory.

**Comment by FagnerMartinsBrack** [comment]
Mouse wheel events were reject in the past, see [this comment](https://github.com/impress/impress.js/pull/51#issuecomment-3507329). We plan to provide a better API so that one can implement this in a per application basis, instead of forcing everyone to have this feature that can have unexpected edge cases.

**Comment by FagnerMartinsBrack** [comment]
Closing due to the lack of feedback from [the OP](http://netforbeginners.about.com/od/internetlanguage/f/What-Is-OP.htm). If you want to work in the PR again, feel free to comment here to open up the discussion again

**Pull request closed without merging**